src/gui/src/SoundLibrary/SoundLibraryExportDialog.cpp

```diff
@@ -175,6 +232,32 @@ void SoundLibraryExportDialog::on_browseBtn_clicked()
 	}
 }
 
+void SoundLibraryExportDialog::on_cancelBtn_clicked()
+{
+	accept();
+}
+
+void SoundLibraryExportDialog::on_drumkitList_currentIndexChanged( QString str )
+{
+	componentList->clear();
+
+	QStringList p_compoList = kit_components[str];
+
+	for (QStringList::iterator it = p_compoList.begin() ; it != p_compoList.end(); ++it) {
+		QString p_compoName = *it;
+
+		componentList->addItem( p_compoName );
+	}
+}
+
+void SoundLibraryExportDialog::on_versionList_currentIndexChanged( QString str )
+{
+	if( str.toStdString() == "0.9.7 and higher" )
```

Comment:
Those strings should be made translatable (in the gui..). So it would be better to use trUtf8() or to use the combo box indices for comparisons.